translations/it_IT.ts

```diff
@@ -19,11 +19,11 @@
     </message>
     <message>
         <source>Contributors</source>
-        <translation>Contribuitori</translation>
+        <translation>Contribuitori:albano battistella & Artyom Zorin</translation>
```

Comment:
Please translate exactly as this string is used as a tab title in the about dialog.